tools/setup/emoji/build_emoji

```diff
@@ -358,5 +342,27 @@ def dump_emojis(cache_path):
     codepoint_to_name_file.write(ujson.dumps(codepoint_to_name))
     codepoint_to_name_file.close()
 
+def dump_emojis(cache_path):
+    # type: (str) -> None
+    with open('emoji_map.json') as emoji_map_file:
+        emoji_map = ujson.load(emoji_map_file)
+
+    EMOJI_DATA_FILE_PATH = os.path.join(EMOJI_DATA_PATH, 'emoji.json')
+    with open(EMOJI_DATA_FILE_PATH) as emoji_data_file:
+        emoji_data = ujson.load(emoji_data_file)
+    emoji_catalog = generate_emoji_catalog(emoji_data)
+
+    UNIFIED_REACTIONS_PATH = os.path.join(ZULIP_PATH, 'zerver', 'management', 'data', 'unified_reactions.json')
+    with open(UNIFIED_REACTIONS_PATH) as unified_reactions_file:
+        unified_reactions_data = ujson.load(unified_reactions_file)
+
+    # Setup emoji farms.
+    run(['rm', '-rf', cache_path])
+    setup_emoji_farm(cache_path, emoji_map, emoji_data)
+    setup_old_emoji_farm(cache_path, emoji_map)
+
+    # Generate various map files.
+    generate_map_files(cache_path, emoji_map, emoji_data, emoji_catalog, unified_reactions_data)
```

Comment:
If easy, can you move this function down in a separate commit, so that the diffs are easier to read?